### PR TITLE
all styling updated

### PR DESCRIPTION
**Pull request by mrogers23458**
hard copied styling from landing-page CSS. Made minor adjustments to index.html, and main.html in order to use some of the CSS styling. All things operational on local branch.

### CODE REVIEW

File: assets/css/holding.css [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+.usedIngredient {
+    color: black;
+    font-family: 'Hamlet';
+}
+
+.missIngredient {
+    color: black;
+    font-style: italic;
+    text-decoration: underline;
+}
```
Comment (Myuze) [inline assets/css/holding.css]:
```suggestion
}

```

File: assets/css/style.css [hunk 2]
```diff
@@ -2,6 +2,14 @@
     box-sizing: border-box;
     margin: 0;
     padding: 0;
+
+
```
Comment (Myuze) [inline assets/css/style.css:6]:
```suggestion
```

File: assets/css/style.css [hunk 3]
```diff
@@ -10,28 +18,109 @@
...
+}
+
+.h1:hover {
+    background-color: rgba(161, 159, 159, 0.911);
+    border-radius: 5px;
+    border: solid 1px grey;
+    box-shadow: inset 90px rgb(82, 81, 81);
+}
+
+
```
Comment (Myuze) [inline assets/css/style.css:49]:
```suggestion
```

File: assets/css/style.css [hunk 4]
```diff
@@ -10,28 +18,109 @@
...
+.h3:hover {
+  color: rgba(161, 159, 159, 0.63);
+    
+}
+
+h3 {
+    color: black;
+}
+
+
```
Comment (Myuze) [inline assets/css/style.css:62]:
```suggestion
```

File: assets/css/style.css [hunk 5]
```diff
@@ -10,28 +18,109 @@
...
+.leader:hover {
+    background-color: rgba(161, 159, 159, 0.911);
+    border-radius: 5px;
+    border: solid 1px grey;
+    box-shadow: inset 90px rgb(82, 81, 81);
+}
+
```
Comment (Myuze) [inline assets/css/style.css:120]:
```suggestion
```

File: assets/css/style.css [hunk 6]
```diff
@@ -168,24 +281,28 @@ with what spoontacular gives us. Selectors for applying styles are important */
...
+    font-family: 'Hamlet';
 }
 
 .missIngredient {
-    color: red;
+    color: black;
     font-style: italic;
     text-decoration: underline;
-}
+}
```
Comment (Myuze) [inline assets/css/style.css]:
```suggestion
}

```

Comment (ghost) [inline assets/css/style.css]: Lost the flex box on the recipe card header
`#cardHeader {
display: flex;
}`

File: docs/index.html [hunk 7]
```diff
@@ -25,14 +25,16 @@
         </div>
     </div>
     <div class="main-container">
+      <div class="cardHolder">
         <p class="h1 text-center">Recipe Champion</p>
-        <p id="lead" class="lead text-center">Enter your ingredients and we'll help you find the perfect recipe</p>
+        <p id="lead" class="leader text-center">Enter your ingredients and we'll help you find the perfect recipe</p>
```
Comment (Myuze) [inline docs/index.html:30]: Is there a specific reason to change this?
The joke prepends to "lead" so either update the .js to match or keep it lead.

**Review by Myuze — CHANGES_REQUESTED** [review]
A few suggestions before commiting.

File: docs/index.html (hunk 7 above)
Comment (mrogers23458) [inline docs/index.html:30]: the id lead and class lead were conflicting. I'll update JS

**Review by Myuze — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**